recipes/pairix/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.1.5" %}
+
+package:
+  name: pairix
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/4dn-dcic/pairix/archive/{{ version }}.tar.gz
+  md5: 05ac0be28a85e4d6bdca744ddec8824e
+
+build:
+  number: 0
+  
+requirements:
+  build:
+    - gcc
+    - zlib
+    - python
+    - setuptools
+
+  run:
+    - python
+    - htslib
```

Comment:
Mh, is zlib really needed as build dependency if you don't have it in the run dependency?